yarn.lock

```diff
@@ -3053,7 +3053,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@carbon/telemetry@npm:0.1.0, @carbon/telemetry@npm:^0.1.0":
+"@carbon/telemetry@npm:0.1.0":
```

Comment:
I got confused at first as to why this was still in the lockfile after removing the dep, but it looks like the versions of other carbon packages are still relying on this so I think it makes sense.